api/migrations/20190715111111_create_sessions_table.sql

```diff
@@ -0,0 +1,17 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+-- +goose StatementBegin
+CREATE TABLE sessions(
+    session_key                 text PRIMARY KEY,
```

Comment:
Tabs/spaces mixup here

Reply:
weird! tab usually only inserts spaces in VSCode. I'll double check my settings & override whatever weird behavior installing Go tools added. 🤷‍♂

Reply:
ah, it's because there were tabs in the file I copied & VSCode tries to match whatever's already there. -_-